<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="WSW-N022" failures="4" tests="4" name="Surefire test" time="2.503" errors="0" timestamp="2021-03-05T01:47:28 CET">
  <testcase classname="tests.Tests" name="exercise1" time="0.003">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at tests.Tests.exercise1(Tests.java:14)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1507)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- exercise1 -->
  <testcase classname="tests.Tests" name="exercise2" time="0.002">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at tests.Tests.exercise2(Tests.java:26)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1507)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- exercise2 -->
  <testcase classname="tests.Tests" name="exercise3" time="0.001">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at tests.Tests.exercise3(Tests.java:36)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1507)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- exercise3 -->
  <testcase classname="tests.Tests" name="exercise4" time="0.003">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at tests.Tests.exercise4(Tests.java:45)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1507)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- exercise4 -->
</testsuite> <!-- Surefire test -->
